service/build.gradle

```diff
@@ -105,6 +105,9 @@ dependencies {
         exclude group: "org.junit.vintage", module: "junit-vintage-engine"
     }
 
+    // Cloud SQL
+    implementation "com.google.cloud.sql:postgres-socket-factory:1.7.0"
```

Comment:
Could you pull this dependency out into a separate PR please? I want to make sure we do some testing on our end before merging, and don't want to block the publishing change you made.

Reply:
removed this implementation from this pull request